### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by partemen86**
added tests, added file URLs and rearranged components into folders

### CODE REVIEW

File: src/actions/ErrorActions.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ export function handleAPIError(e) {
     return { type: 'AUTH_ERROR' };
   } else if (e.response.status === 404) {
     console.log('Resource not found');
```
Comment (partemen86) [inline src/actions/ErrorActions.js:5]: fixed actions to return an object to avoid console errors

File: src/components/Files/File.js [hunk 2]
```diff
@@ -4,18 +4,25 @@ const File = ({file, index, handleDelete}) => {
...
+      <div className='flex-none flex flex-row items-center px-8 w-1/4'>
         <p className='flex-none text-gray-600 text-md'>Last update:</p>
-        <p className='flex-auto text-bdazzledblue text-md font-light ml-2'>{instanceCreationDate.toLocaleString()}</p>
+        <p className='whitespace-nowrap overflow-auto flex-auto text-bdazzledblue text-md font-light ml-2'>{instanceCreationDate.toLocaleString()}</p>
       </div>
-      <div id='deleteFile' className='flex-auto flex flex-col items-end py-4 pr-4 stroke-redorange' onClick={() => handleDelete(file.id)}>
+      <div id='deleteFile' className='flex-auto flex flex-col items-end pr-4 stroke-redorange' onClick={() => handleDelete(file.id)}>
         <svg xmlns="http://www.w3.org/2000/svg" className='hover:cursor-pointer' width="20" viewBox="0 0 24 24" fill="none" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
           <circle cx="12" cy="12" r="10"></circle>
           <line x1="15" y1="9" x2="9" y2="15"></line>
```
Comment (partemen86) [inline src/components/Files/File.js:28]: added some css to fix overflows and some of the margins/padding.  Its not perfect but I think its much better this way

File: src/components/Files/Files.js [hunk 3]
```diff
@@ -37,4 +37,4 @@ const Files = () => {
   );
 };
 
-export default Files;
```
Comment (partemen86) [inline src/components/Files/Files.js:40]: made file divs bigger since i added URLs to them

File: src/components/Links/BastionSidebarLink.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      className={'w-full flex flex-col px-4 py-2 h-12 rounded-md text-white2 hover:text-tomato ' + linkCSS}
+    >
+      <div className='flex flex-row items-center'>
+        <svg className="w-6 h-6 mr-2" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M12 6V4m0 2a2 2 0 100 4m0-4a2 2 0 110 4m-6 8a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4m6 6v10m6-2a2 2 0 100-4m0 4a2 2 0 110-4m0 4v2m0-6V4"></path>
+        </svg>
+        <p className='pl-1 pt-1'>{title}</p>
+      </div>
+    </Link>
+  );
```
Comment (partemen86) [inline src/components/Links/BastionSidebarLink.js:19]: this file was just renamed

File: src/components/Users/User.js [hunk 5]
```diff
@@ -7,16 +7,16 @@ const User = ({user, index, handleDelete}) => {
...
         <p className='flex-none text-gray-600 text-md'>Email:</p>
-        <p className='flex-auto text-bdazzledblue text-md ml-2'>{user.email}</p>
+        <p className='whitespace-nowrap overflow-auto flex-auto text-bdazzledblue text-md ml-2'>{user.email}</p>
       </div>
-      <div className='flex-none flex flex-row items-center py-4 px-8 w-1/3'>
+      <div className='flex-none flex flex-row items-center py-4 px-8 w-1/4'>
         <p className='flex-none text-gray-600 text-md'>Last update:</p>
-        <p className='flex-auto text-bdazzledblue text-md font-light ml-2'>{instanceCreationDate.toLocaleString()}</p>
+        <p className='whitespace-nowrap overflow-auto flex-auto text-bdazzledblue text-md font-light ml-2'>{instanceCreationDate.toLocaleString()}</p>
       </div>
```
Comment (partemen86) [inline src/components/Users/User.js:20]: fixed overflow issues, changed name to username

**Review by Reillyknut — APPROVED** [review]
Re-deployed and everything was working smoothly. Changes look good. Tests all work. New admin app image is 0.17.12

**Pull request merged**